ci/ci

```diff
@@ -647,7 +647,7 @@ def setup_config_dir():
         write_to_file(target_path, value)
     target_path = os.path.join(command_path, MACROS['BENCHPRESS_OPT_FILE'])
     if not os.path.exists(target_path):
-        value = "--tags=smoke"
+        value = "-t SmokeTest"
```

Comment:
Why did we change to smoke test instead of smoke tag ?

Reply:
The default test to run in ci docs is mentioned as SmokeTest. This was erroneous in previous check in. We don't want run with tag... Because the requirement decorator will be generated with multiple nodes and we don't want to run a multi node test just for smoke.